configuration/scripts/tests/create_fails.csh

```diff
@@ -0,0 +1,24 @@
+#!/bin/csh
+
+echo " "
+set tmpfile = create_fails.tmp
+set outfile = fails.ts
+
+./results.csh >& /dev/null
+cat results.log | grep ' run\| test' | grep -v PASS | cut -f 2 -d " " | sort -u >! $tmpfile
+
+echo "# Test  Grid  PEs  Sets" >! fails.ts
+foreach line ( "`cat $tmpfile`" )
+  #echo $line
+  set test = `echo $line | cut -d "_" -f 3`
+  set grid = `echo $line | cut -d "_" -f 4`
+  set pes  = `echo $line | cut -d "_" -f 5`
+  set opts = `echo $line | cut -d "_" -f 6- | sed 's/_/,/g'`
+  echo "$test  $grid  $pes  $opts" >> fails.ts
+end
+
+rm $tmpfile
+echo "$0 done"
+echo "Not passed tests written to file...... $outfile"
```

Comment:
Maybe "Failed tests" instead ?

Reply:
Failed tests isn't quite accurate.   Failed != Not passed.  But I think you're right on reflection that failed is probably a better word here.  I'll change it.